cg/apps/orderform/excel_orderform_parser.py

```diff
@@ -175,20 +175,13 @@ def parse_data_analysis(self) -> str:
         return data_analyses.pop().lower().replace(" ", "-")
 
     def get_data_delivery(self) -> str:
-        """Determine the order_data delivery type"""
-
-        data_delivery: str = self.parse_data_delivery()
-
-        try:
-            return DataDelivery(data_delivery)
-        except ValueError as error:
-            raise OrderFormError(f"Unsupported Data Delivery: {data_delivery}") from error
+        """Determine the order_data delivery type."""
+        return self.parse_data_delivery()
```

Comment:
I feel like `parse_data_delivery` can just replace `get_data_delivery` since the later does not do anything other than call the first